public/app/plugins/panel/geomap/editor/StyleEditor.tsx

```diff
@@ -200,24 +216,48 @@ export const StyleEditor = (props: Props) => {
         />
       </Field>
       {!settings?.hideSymbol && (
-        <Field label={'Symbol'}>
-          <ResourceDimensionEditor
-            value={value?.symbol ?? defaultStyleConfig.symbol}
-            context={context}
-            onChange={onSymbolChange}
-            item={
-              {
-                settings: {
-                  resourceType: MediaType.Icon,
-                  folderName: ResourceFolderName.Marker,
-                  placeholderText: hasTextLabel ? 'Select a symbol' : 'Select a symbol or add a text label',
-                  placeholderValue: defaultStyleConfig.symbol.fixed,
-                  showSourceRadio: false,
-                },
-              } as StandardEditorsRegistryItem
-            }
-          />
-        </Field>
+        <>
+          <Field label={'Symbol'}>
+            <ResourceDimensionEditor
+              value={value?.symbol ?? defaultStyleConfig.symbol}
+              context={context}
+              onChange={onSymbolChange}
+              item={
+                {
+                  settings: {
+                    resourceType: MediaType.Icon,
+                    folderName: ResourceFolderName.Marker,
+                    placeholderText: hasTextLabel ? 'Select a symbol' : 'Select a symbol or add a text label',
+                    placeholderValue: defaultStyleConfig.symbol.fixed,
+                    showSourceRadio: false,
+                  },
+                } as StandardEditorsRegistryItem
+              }
+            />
+          </Field>
+          <Field label={'Anchor Position'}>
```

Comment:
tiny nits: I think there might be a component that we can use to group these two together and add the label at that level vs. on just the y anchor field

or maybe explicetely add labels to both (i.e. `Anchor Y Position` etc)

Also not sure on the wording of "anchor" vs just using "symbol" (i.e. `Symbol Position`) this makes more sense to me as user is determining behavior of the symbol aspect of the marker 🤷‍♂️